pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -73,18 +71,15 @@ const (
 	}
 }
 `
-	//TODO: temporary until fix org isolation
-	mainOrgID = 1
 )
 
 type Alertmanager struct {
 	logger      log.Logger
 	gokitLogger gokit_log.Logger
 
-	Settings *setting.Cfg       `inject:""`
-	SQLStore *sqlstore.SQLStore `inject:""`
+	Settings *setting.Cfg
 	Store    store.AlertingStore
-	Metrics  *metrics.Metrics `inject:""`
```

Comment:
Just to make sure that removing the `inject` annotation is expected?

Reply:
We are. We now need to control the metric registration on the `multiorg` struct so that we can keep track of created/deleted registries. Injecting the metrics struct directly doesn't make sense outside of the controlling struct.